x/sisu/keeper/keeper.go

```diff
@@ -408,6 +419,19 @@ func (k *DefaultKeeper) GetTransferQueue(ctx sdk.Context, chain string) []*types
 	return getTransferQueue(queueStore, transferStore, chain)
 }
 
+///// Failed Transfer Queue
+func (k *DefaultKeeper) AddFailedTransfer(ctx sdk.Context, transferId string) int64 {
```

Comment:
Where do you get the failed transfer & where do you remove it? (Assume that the retry succeeds)